src/current/cockroachcloud/cloud-api.md

```diff
@@ -152,138 +167,267 @@ curl --request POST \
 
 {% include_cached copy-clipboard.html %}
 ~~~ json
 {
-  "cloud_provider": "{cloud_provider}",
-  "created_at": "2022-03-14T14:15:22Z",
-  "creator_id": "{account_id}",
-  "deleted_at": "2022-03-14T14:15:22Z",
-  "id": "{cluster_id}",
-  "operation_status": "CLUSTER_STATUS_UNSPECIFIED",
   "name": "{cluster_name}",
-  "plan": "SERVERLESS",
-  "regions": [
-    {
-      "name": "{region_name}",
-      "sql_dns": "{server_host}",
-      "ui_dns": ""
-    }
-  ],
-  "config": {
+  "provider": "{cloud_provider}",
+  "plan": "STANDARD",
+  "spec": {
     "serverless": {
       "regions": [
         "{region_name}"
       ],
-      "spend_limit": 0,
-      "routing_id": "{routing_id}"
+      "usage_limits": {
+        "provisioned_virtual_cpus": "2"
+      }
     }
-  },
-  "state": "CREATING",
-  "updated_at": "2022-03-14T14:15:22Z"
+  }
 }
 ~~~
 
+</section>
+
 Where:
 
-  - `{cloud_provider}` is the name of the cloud infrastructure provider on which you want your cluster to run. Possible values are: `GCP`, `AWS`, `AZURE`.
-  - `{cluster_id}` is the unique ID of this cluster. Use this ID when making API requests for this particular cluster.
-    {{site.data.alerts.callout_info}}
-    The cluster ID used in the Cloud API is different from the routing ID used when [connecting to clusters]({% link cockroachcloud/connect-to-a-basic-cluster.md %}).
-    {{site.data.alerts.end}}
-  - `{cluster_name}` is the name of the cluster you specified when creating the cluster.
-  - `{account_id}` is the ID of the account that created the cluster. If the cluster was created using the API, this will be the service account ID associated with the secret key used when creating the cluster.
-  - `{region_name}` is the zone code of the cloud infrastructure provider where the cluster is located.
-  - `{routing_id}` is the cluster name and tenant ID of the cluster used when [connecting to clusters]({% link cockroachcloud/connect-to-a-basic-cluster.md %}). For example, `funky-skunk-123`.
-  - `{server_host}` is the DNS name of the host on which the cluster is located.
+  - `{cluster_name}` is the name of the cluster. The name must be 6-20 characters in length and can include numbers, lowercase letters, and dashes (but no leading or trailing dashes).
+  - `{cloud_provider}` is the name of the cloud provider on which you want your cluster to run: `AWS`, `AZURE`, or `GCP`.
+  - `{region_name}` is the name of a CockroachDB Cloud [region]({% link cockroachcloud/regions.md %}). Region names are set by the cloud provider. For example,`us-west2` is a GCP region. Available regions vary based on both the selected plan type (`BASIC`, `STANDARD`, or `ADVANCED`) and the cloud provider.
+  - `plan` is the cluster's plan, `BASIC`, `STANDARD`, or `ADVANCED`. The default is `STANDARD`.
+  - The `usage_limits` field specifies the resource limits for the cluster. The `provisioned_virtual_cpus` field indicates the maximum number of virtual CPUs (vCPUs) the cluster can provision.
```

Comment:
I think a bit better would be:
"the number of virtual CPUs (vCPUs) the cluster provides."

Reply:
Updated to use this language. Thanks.